addons/website/migrations/13.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,139 @@
+---Models in module 'website'---
+obsolete model website.redirect (renamed to website.rewrite)
+new model website.rewrite (renamed from website.redirect)
+# DONE: pre-migration: renamed models
+
+new model website.route
+new model website.track
+new model website.visitor
+# NOTHING TO DO
+
+---Fields in module 'website'---
+website      / ir.ui.view               / track (boolean)               : NEW hasdefault
+website      / mrp.document             / key (char)                    : previously in module mrp

```

Comment:
This is for sure a bug in the `openupgrade_records` module, as there's no `mrp.document` here in this module. Can you please check to see the problem and see if this is important.